app/helpers/evms_helper.rb

```diff
@@ -34,20 +44,27 @@ def cr_color
     value.html_safe
   end
 
+  # Get project name
+  #
+  # @return [String] project name, baseline subject
   def project_chart_name
-    unless @baseline_id.nil?
-      @project.name + ' - ' + @evmbaseline.find(@baseline_id).subject
-    else
-      @project.name
-    end
+    @baseline_id.nil? ? @project.name : @project.name + ' - ' + @evmbaseline.find(@baseline_id).subject
```

Comment:
Line is too long. [103/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.